etl/steps/data/garden/tuberculosis/2023-11-27/drug_resistance_surveillance.meta.yml

```diff
@@ -0,0 +1,236 @@
+      rr_15plus:
+        title: Bacteriologically confirmed pulmonary TB patients with rifampicin resistance who are aged 15 years and over
+        unit: patients
+        description_short: Number of bacteriologically confirmed pulmonary TB patients with rifampicin resistance who are aged 15 years and over.
+        description_from_producer: Among new and previously treated patients that have confirmed pulmonary TB cases with results for rifampicin testing, number resistant to rifampicin and who are aged 15 years and over.
+      rr_ageunk:
+        title: Bacteriologically confirmed pulmonary TB patients with rifampicin resistance whose age is unknown
+        unit: patients
+        description_short: Number of bacteriologically confirmed pulmonary TB patients with rifampicin resistance whose age is unknown.
+        description_from_producer: Among new and previously treated patients that have confirmed pulmonary TB cases with results for rifampicin testing, number resistant to rifampicin and whose age is unknown
+      nrr_014:
+        title: Bacteriologically confirmed pulmonary TB patients with rifampicin susceptibility who are aged 0-14 years
+        unit: patients
+        description_short: Number of bacteriologically confirmed pulmonary TB patients with rifampicin susceptibility who are aged 0-14 years.
+        description_from_producer: Among new and previously treated patients that have confirmed pulmonary TB cases with results for rifampicin testing, number susceptible to rifampicin and who are aged 0-14 years
+      nrr_15plus:
+        title: Bacteriologically confirmed pulmonary TB patients with rifampicin susceptibility who are aged 15 years and over
+        unit: patients
+        description_short: Number of bacteriologically confirmed pulmonary TB patients with rifampicin susceptibility who are aged 15 years and over.
+        description_from_producer: Among new and previously treated patients that have confirmed pulmonary TB cases with results for rifampicin testing, number susceptible to rifampicin and who are aged 15 years and over
+      nrr_ageunk:
+        title: Bacteriologically confirmed pulmonary TB patients with rifampicin susceptibility whose age is unknown
+        unit: patients
+        description_short: Number of bacteriologically confirmed pulmonary TB patients with rifampicin susceptibility whose age is unknown.
+        description_from_producer: Among new and previously treated patients that have confirmed pulmonary TB cases with results for rifampicin testing, number susceptible to rifampicin and whose age is unknown
+      rr_dst_rlt:
+        title: Bacteriologically confirmed pulmonary TB patients with rifampicin resistance tested for any fluoroquinolone and any second-line injectable agent
+        unit: patients
```

Comment:
Here a description_short might be needed to explain what fluoroquinolone is/does